### ISSUE TITLE
Remove Mastodon links (website, GitHub, ..)

### ISSUE THREAD

**Issue by berendt** [SovereignCloudStack/issues#497]
It looks as if Mastodon is not being actively used. Accordingly, the links should be removed. The pinned post on Mastodon is the 4th release from SCS. There is already the 5th release, which is confusing if the account is not active and contains old information but is actively linked.

**Comment by fkr** [comment]
I agree. However I do think, this is less than ideal, since basically only linkedin is being is currently covered with some SCS-related posts. I'll bring this up in the SCS-project team.
